tests-node/unit/broccoli/translation-reducer/index-test.js

```diff
@@ -36,15 +36,15 @@ describe('translation-reducer', function () {
       });
 
       this.input.write({
-        'en-us.json': `{ "greet": "hi" }`,
+        'en-us.json': `{ "greet": "你好" }`,
       });
 
       return build(createBuilder(subject), (fs) => {
         expect(fs).to.deep.equal({
           custom: {
             output: {
               path: {
-                'en-us.json': `{"greet":"hi"}`,
+                'en-us.json': `{"greet":"你好"}`,
```

Comment:
Can we revert this and line 39.  Other than that, looks good to go!  Thanks

Reply:
😆，sorry， i mad a mistake when replace txt